pkg/kubectl/cmd/get.go

```diff
@@ -220,25 +221,56 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 		return printer.PrintObj(obj, out)
 	}
 
+	objs := []runtime.Object{}
+	mappings := []*meta.RESTMapping{}
+	err = b.Flatten().Do().Visit(func(r *resource.Info, err error) error {
+		if err != nil {
```

Comment:
It would be better to call Infos() and Objects() to build these lists for you (keep the result of Do() which caches them).